specifications/sep-payment-channel-mechanism.md

```diff
@@ -0,0 +1,631 @@
+account E, and deposits R's reserves to reserve account V. TA_i has any source
+account that is not E or V, typically the participant proposing the change.
+
+  TA_i contains operations:
+
+  - Operations sponsored by I:
+    - One `BEGIN_SPONSORING_FUTURE_RESERVES` operation that specifies
+    participant I as a sponsor of future reserves.
+    - One or more `CHANGE_TRUST` operations adding trustlines to E.
+    - One `END_SPONSORING_FUTURE_RESERVES` operation that stops I sponsoring
+    future reserves of subsequent operations.
+  - One or more `PAYMENT` operations depositing R's reserves to V, for each new
+  trustline on E that will be used to sponsor claimable balances at
+  disbursement.
+
+##### Remove Trustline
+
+Participants can remove empty trustlines.
+
+1. I and R sign and exchange signatures for trustline transaction TR_i.
+2. I or R submit TR_i.
+
+If the remove trustline transaction TR_i fails or is never submitted, there is
+no consequence to the channel.
+
+The transactions are constructed as follows:
+
+- TR_i, the _remove trustline transaction_, removes one or more trustline on
+escrow account E, and withdraws R's reserves from reserve account V. TR_i has
+any source account that is not E or V, typically the participant proposing the
+change.
+
+  TR_i contains operations:
+
+  - Operations sponsored by I:
+    - One `BEGIN_SPONSORING_FUTURE_RESERVES` operation that specifies
+    participant I as a sponsor of future reserves.
+    - One or more `CHANGE_TRUST` operations removing trustlines from E.
+    - One `END_SPONSORING_FUTURE_RESERVES` operation that stops I sponsoring
+    future reserves of subsequent operations.
+  - One or more `PAYMENT` operations withdrawing R's reserves from V, for each
+  trustline being removed from E that would have been used to sponsor claimable
+  balances at disbursement and are no longer required.
+
+##### Deposit by Initiator
+
+Participant I may deposit into the channel without coordination with
+participant R, as long as escrow account E already has a trustline for the
+asset being deposited.
+
+If participant I wishes to deposit an asset that escrow account E does not hold
+a trustline for, the [Add Trustlines](#Add-Trustline) process must be used
+first.
+
+##### Deposit by Responder
+
+Participant R may deposit into the channel without coordination with participant
+I, as long as escrow account E already has a trustline for the asset being
+deposited, and as long as participants R's intent is to make a payment of the
+same value to participant I. Any amounts deposited to the payment channel
+without coordination will be disbursable to participant I at close.
+
+Participant R must coordinate with participant I to deposit any amount that it
+does not intend to immediately pay participant I. The participants use the
+following process:
+
+1. Increment i.
+2. I and R build the deposit transaction P_i.
+3. Set e' to e.
```

Comment:
It took me a while to realize `e'` is an auxiliary var you're using, since all the other vars are well-defined concepts here. Maybe change the phrasing to `Store the current value for e on e'`?

Reply:
```suggestion
3. Set e' to the value of e.
```

Reply:
Ah yes this is not particularly clear. Thanks. I used slightly different language in ff8ec83.